### ISSUE TITLE
Protocol 

### ISSUE THREAD

**Issue by jjehl** [SkRobo/rplidar#6]
lots of command raise issues. For example, get_info() after a scan is wrong. I will do some changes in the code to make it more reliable.
Are you interested I make a pull request (this means, you are always active on this project !) or do you prefer I improve only my fork ?

**Comment by newpavlov** [comment]
Yes, I am interested and I will be grateful for your contribution!
